docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,134 @@
+* <<apm-otel-api-sdk-elastic-agent,OpenTelemetry API/SDK + Elastic APM agent>>
+* <<apm-otel-native-support,OpenTelemetry native support>>
+* <<apm-otel-lambda,Lambda collector exporter>>
 
-To unlock the full power of the {stack}, use the OpenTelemetry API/SDKs with Elastic APM agents,
-currently supported by the Java, Python, .NET, and Node.js agents.
-These Elastic APM agents translate OpenTelemetry API calls to Elastic APM API calls.
-This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--
-avoiding vendor lock-in and having to redo manual instrumentation.
+[float]
+[[apm-otel-elastic-distros]]
+== Elastic Distributions for OpenTelemetry Language SDKs
 
-<<apm-open-telemetry-with-elastic,Get started>>.
+preview::[]
 
-**OpenTelemetry agent**
+Elastic offers several distributions for OpenTelemetry language SDKs.
+A _distribution_ is a customized version of an upstream OpenTelemetry repository.
+Each Elastic Distribution for OpenTelemetry is a wrapper around an https://opentelemetry.io/docs/languages/[OpenTelemetry Language SDK].
 
-The {stack} natively supports the OpenTelemetry protocol (OTLP).
-This means trace data and metrics collected from your applications and infrastructure by an
-OpenTelemetry agent can be sent directly to the {stack}.
+image::images/apm-otel-distro.png[]
+
+With an Elastic Distribution for OpenTelemetry Language SDK you have access to all the features of the OpenTelemetry SDK that it customizes, plus:
+
+* You may get access to SDK improvements and bug fixes contributed by the Elastic team _before_
+the changes are available upstream in the OpenTelemetry repositories.
+* The distro preconfigures the collection of tracing and metrics signals, applying some opinionated defaults,
+such as which sources are collected by default.
+
+// Why you wouldn't choose this method
+// Still in tech preview?
+
+Get started with an Elastic Distribution for OpenTelemetry Language SDK:
+
+* preview:["The Elastic Distribution for OpenTelemetry Java is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-java[*Elastic Distribution for OpenTelemetry Java →*]
+* preview:["The Elastic Distribution for OpenTelemetry .NET is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-dotnet[*Elastic Distribution for OpenTelemetry .NET →*]
+* preview:["The Elastic Distribution for OpenTelemetry Node.js is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-node[*Elastic Distribution for OpenTelemetry Node.js →*]
+
```

Comment:
We also have a distribution in `technical preview` for Python, that we should add in this list:
https://github.com/elastic/elastic-otel-python